targets/android-studio

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+REQUIRES='xiwi gtk-extra'
+DESCRIPTION='Installs Android Studio as a directly launchable xiwi application'
+HOSTBIN='startandroidstudio'
+
+### Append to prepare.sh:
+# Must create the /etc/bash_completion.d directory
+# in order for umake to properly install
+mkdir /etc/bash_completion.d
```

Comment:
Must be specific to yakkety/zesty then — get a "Directory non-existent" error when trying to configure in those cases.